content/docs/getting-started/db.md

```diff
@@ -285,15 +318,15 @@ use tokio_minihttp::{Request, Response};
 use tokio_proto::TcpServer;
 use tokio_service::Service;
 
-struct Server {
-    thread_pool: CpuPool,
-    db_pool: r2d2::Pool<r2d2_postgres::PostgresConnectionManager>,
-}
-
 #[derive(Serialize)]
 struct Message {
     id: i32,
-    randomNumber: i32,
+    body: String,
+}
+
+struct Server {
```

Comment:
I switched the order of the `Message` struct and the `Server` struct so that the `Server` would be defined right before it's `impl`.